src/example/example.service.ts

```diff
@@ -0,0 +1,93 @@
+export class ExampleService {
+  public calculateAverage(numbers: number[]): number {
+    if (!numbers.length) return 0;
+    let sum = 0;
+    for (const num of numbers) {
+      sum += num;
+    }
+    return sum / numbers.length;
```

Comment:
PullProofAI

The `calculateAverage` method in the `ExampleService` class lacks error handling for the case where `numbers` is an empty array. Consider throwing an error or returning a meaningful value in such cases.



<!-- This is an auto-generated comment by PullProofAI -->